changelogs/fragments/79657-warn-for-unsupported-role-arg-spec-fields.yml

```
@@ -0,0 +1,2 @@
+bugfixes:
+  - roles - give a warning for unknown fields during implicit M(validate_argument_spec) tasks.
```

```suggestion
  - roles - give a warning for unknown fields during implicit ``validate_argument_spec`` tasks.
```
`M(...)` only works in documentation, not in changelogs. (And even if it would, you'd need `M(ansible.builtin.validate_argument_spec)` here.)